lighthouse-core/report/html/report-styles.css

```diff
@@ -537,6 +537,26 @@
   display: block;
 }
 
+
+.lh-filterbar {
```

Comment:
filterbar -> metricfilter or metricfilterbar or something. We have a 3p filter so "filter" is too ambiguous.